pkg/workflows/sdk/v2/trigger.go

```diff
@@ -0,0 +1,22 @@
+package sdk
+
+import (
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/types/known/anypb"
+)
+
+type Trigger[T proto.Message] interface {
+	NewT() T
+	Id() string
```

Comment:
```suggestion
        // Id uniquely identifies a Trigger (something more?)
	Id() string
```

Reply:
Id is pretty self-describing. Not everything needs a comment, and the interface does not provide methods that a workflow author is expected to call.